lighthouse-core/test/audits/is-on-https-test.js

```diff
@@ -55,6 +57,24 @@ describe('Security: HTTPS audit', () => {
     });
   });
 
+  it('augmented with mixed-content InspectorIssues', async () => {
+    const networkRecords = [
+      {url: 'https://google.com/', parsedURL: {scheme: 'https', host: 'google.com'}},
+      {url: 'http://localhost/image.jpeg', parsedURL: {scheme: 'http', host: 'localhost'}},
+      {url: 'https://google.com/', parsedURL: {scheme: 'https', host: 'google.com'}},
+    ];
+    const mixedContentIssues = [
+      {insecureURL: 'http://localhost/image.jpeg', resolutionStatus: 'MixedContentBlocked'},
+    ];
+    const artifacts = getArtifacts(networkRecords, mixedContentIssues);
+    const result = await Audit.audit(artifacts, {computedCache: new Map()});
+
+    expect(result.details.headings).toHaveLength(2);
```

Comment:
if keeping the `if (artifacts.InspectorIssues.mixedContent.length)` check, should probably add a test for that case too (the above only assert `extendedInfo.value.length`)